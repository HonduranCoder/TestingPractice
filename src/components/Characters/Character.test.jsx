import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Character from './Character.jsx';

it('should render Character', () => {
  const character = {
    image: 'https://rickandmortyapi.com/api/location/1',
    name: 'Rick',
    species: 'Human',
    status: 'Alive',
  };
  const { container } = render(
    <MemoryRouter>
      <Character character={character} />
    </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
