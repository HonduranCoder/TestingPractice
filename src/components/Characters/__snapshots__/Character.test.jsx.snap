// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Character 1`] = `
<div>
  <figure>
    <img
      alt="Rick"
      src="https://rickandmortyapi.com/api/location/1"
    />
    <figcaption>
      <h3>
        Rick
      </h3>
      <p>
        Human
      </p>
      <p>
        Status: 
        Alive
      </p>
    </figcaption>
  </figure>
</div>
`;
